content/docs/2.7/scalers/gcp-storage.md

```diff
@@ -0,0 +1,122 @@
++++
+title = "Google Cloud Platform Storage"
+availability = "2.7+"
+maintainer = "Community"
+description = "Scale applications based on the count of objects in a given GCP Storage bucket."
```

Comment:
```suggestion
description = "Scale applications based on the count of objects in a given Google Cloud Storage (GCS) bucket."
```

Reply:
Yes, I like this more and would use this in all places `Google Cloud Storage (GCS)`